docs/frontity-cli/dev.md

```diff
@@ -19,6 +19,25 @@ npx frontity dev [options]
 |     `--target <target>`      | create bundles with "es5" or "module". Default target is "module".                                                                 |
 | `-h`, `--help`  | Output usage information                                                                                                             |
 
+##### The `--production` option
+
+This flag correspond to [webpack’s mode parameter](https://webpack.js.org/configuration/mode/) so it will run webpack in the production mode as described [there](https://webpack.js.org/configuration/mode/) before launching the development server.
+
+So, if you do:
+
+```
+npx frontity dev --production
+```
+
+The webpack bundler internally will:
+
+- Enable certain webpack-specific optimizations and minify the code
+- Also disable hot-module reloading (HMR)
+- Not create source maps
+- Append hashes to filenames so for caching purposes (more info on the how create-react-app does it)
+
+Normally, you would always use the development server in development mode, but sometimes you may want to check that everything works in production mode, or check the bundle analyzer (the files at `/build/analyze`) for the production bundle. You can do so by using `npx frontity dev --production`
```

Comment:
The last phrase `You can do so by using npx frontity dev --production` feels repetitive.